tests/unit/test_slurm_utils.py

```diff
@@ -26,6 +27,15 @@
 IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
 
 
+def test_is_on_slurm():
```

Comment:
Oh I see, this is just merge conflict stuff - it's fine as is and I'll get the better `is_on_slurm` in my pr